CHANGELOG.md

```diff
@@ -14,11 +14,35 @@
 - Adds the `ScreenshotButton` component that takes a screenshot ([#4714](https://github.com/getsentry/sentry-react-native/issues/4714))
 - Add Feedback Widget theming ([#4677](https://github.com/getsentry/sentry-react-native/pull/4677))
 - Re-export `ErrorEvent` and `TransactionEvent` types ([#4859](https://github.com/getsentry/sentry-react-native/pull/4859))
+- Add experimental support for Log tracing ([#4827](https://github.com/getsentry/sentry-react-native/pull/4827))
+
+To enable it add the following code to your Sentry Options:
+
+```typescript
+  _experiments: {
+    enableLogs: true,
+  },
+```
+
+You can also filter the logs being collected by adding beforeSendLogs into `_experiments`
+
+```typescript
+  _experiments: {
+    enableLogs: true,
+    beforeSendLog: (log) => {
+      return log;
+    },
+  },
+  ```
```

Comment:
nit: I think we should align one tab to the right

Reply:
I added a full snippet that should fix the issue here https://github.com/getsentry/sentry-react-native/pull/4827/commits/fc591b54523880c4c2bb10bdf6d6e2fa7a3f90ad